src/tree-map.satyg

```diff
@@ -0,0 +1,75 @@
+@stage: persistent
+@import: base
+@import: ord
+@import: tree-set
+@import: option-ext
+@import: int
+@import: stream
+@import: tuple
+
+module TreeMap : sig
+  % Immutable map.
```

Comment:
For consistency with other modules like [Array](https://github.com/nyuichi/satysfi-base/blob/master/src/array.satyg), you may want to start doc comments with `%%....`

Reply:
This also applies to treeset.satyg

Reply:
covered 👍